cypress/integration/transforms_spec.js

```diff
@@ -0,0 +1,238 @@
+    });
+
+    describe("can be created", () => {
+      it("successfully", () => {
+        // Confirm we loaded empty state
+        cy.contains(
+          "Transform jobs help you create a materialized view on top of existing data."
+        );
+
+        // Route to create transform page
+        cy.contains("Create transform").click({ force: true });
+
+        // Type in transform ID
+        cy.get(`input[placeholder="my-transformjob1"]`).type(TRANSFORM_ID, { force: true });
+
+        // Get description input box
+        cy.get(`textarea[data-test-subj="description"]`).focus().type("some description");
+
+        // Enter source index
+        cy.get(`div[data-test-subj="sourceIndexCombobox"]`)
+          .find(`input[data-test-subj="comboBoxSearchInput"]`)
+          .focus()
+          .type("opensearch_dashboards_sample_data_ecommerce{enter}");
+
+        // Enter target index
+        cy.get(`div[data-test-subj="targetIndexCombobox"]`)
+          .find(`input[data-test-subj="comboBoxSearchInput"]`)
+          .focus()
+          .type("target_index{enter}");
+
+        // Click the next button
+        cy.get("button").contains("Next").click({ force: true });
+
+        // Confirm that we got to step 2 of creation page
+        cy.contains("Select fields to transform");
+
+        // Setup Group and aggregation
+        cy.wait(2000);
+        cy.contains("category.keyword").parent().parent().parent().next()
+          .children().first().children().first().children().first()
+          .click({ force: true });
+
+        cy.contains("Group by terms").click({ force: true });
+
+        // Confirm group was added
+        cy.contains("category.keyword_terms");
+
+        // Add aggregable field
+        cy.contains("50 columns hidden").click({ force: true });
+        cy.contains("taxless_total_price").click({ force: true });
+        // Click out of the window
+        cy.contains("Select fields to transform").click({ force: true });
+
+        cy.contains("taxless_total_price").parent().parent().parent().next()
+          .children().first().children().first().children().first()
+          .click({ force: true });
+        cy.contains("Aggregate by avg").click({ force: true });
+
+        // Confirm agg was added
+        cy.contains("avg_taxless_total_price");
+
+        // Click the next button
+        cy.get("button").contains("Next").click({ force: true });
+
+        // Confirm that we got to step 3 of creation page
+        cy.contains("Job enabled by default");
+
+        // Click the next button
+        cy.get("button").contains("Next").click({ force: true });
+
+        // Confirm that we got to step 4 of creation page
+        cy.contains("Review and create");
+
+        // Click the create button
+        cy.get("button").contains("Create").click({ force: true });
+
+        // Verify that sample data is add by checking toast notification
+        cy.contains(`${TRANSFORM_ID}`);
```

Comment:
Can we change this to `cy.contains("Transform job "${TRANSFORM_ID}" successfully created.")` to make sure that it is successfully created (according to [this line of code](https://github.com/opensearch-project/index-management-dashboards-plugin/blob/47e44423f029b9a55674627a38248f9f83e06a25/public/pages/CreateTransform/utils/helpers.ts#L61)? If it turns out to fail the creation, the toast message would also contain the transform id, too. [Code here](https://github.com/opensearch-project/index-management-dashboards-plugin/blob/47e44423f029b9a55674627a38248f9f83e06a25/public/pages/CreateTransform/containers/CreateTransformForm/CreateTransformForm.tsx#L472)